Emilia-Capital/comment-hacks
# Cleanup - prep for new plugin

## Context

Preliminary work for the *new* plugin which will be released with a `comment-hacks` slug

* Remove references and links to Yoast
* Rename the plugin from `yoast-comment-hacks` to `comment-hacks`
* Change repo URLs to point to the new URL under the `Emilia-Capital` umbrella
* Replace the `yoast_` and `yoast-` prefixes with `emiliaprojects_` and `emiliaprojects-`
* Remove deprecated code. Since this will be a new plugin, deprecated code should not be kept
* Change the textdomain from `yoast-comment-hacks` to `comment-hacks`
* Rename global JS functions to use the `emiliaProjects` prefix instead of `yoast`
* Remove global variables containing the `yoast` prefix
* Clean-up the changelog. 

## Test instructions

Test that the main plugin functionality and admin UIs still works as expected. A lot of stuff was renamed, classes, IDs etc, so we should make sure things are still in the right place.
NOTE: Since the main plugin file was renamed, if the previous version is activated, it should be deactivated prior to pulling this PR. This should not be treated as an update, but as a new plugin.

#### Relevant test scenarios

## UI changes

NOTE: The UI itself didn't change, however the HTML of the UI did.

## Documentation


## Quality assurance


Fixes #
